src/test/java/uk/gov/dhsc/htbhf/page/TestNamesPage.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.dhsc.htbhf.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import uk.gov.dhsc.htbhf.page.component.InputField;
+
+/**
+ * Page object for the enter your name page.
+ */
+public class TestNamesPage extends SubmittablePage {
```

Comment:
What is the purpose of this page object, I think if this is to be committed then it needs to have a good description and a better name rather than `TestNames`.

Reply:
the reason i have created this object was the next page after tes/nhs-number is test/names, this is only for specific web-ui-branch. So to cope with this i have created new pageobject.

Reply:
Can the name of this class be a bit more specific? Maybe EnterYourNamePage, also this is already under test package so why prefix with Test?